packages/api/src/task/scheduler.ts

```diff
@@ -428,7 +429,12 @@ export class TaskScheduler {
     };
 
     task = await this.updateTask(task, { status });
-    await sleep(retries * TASK_RETRY_BASE_DELAY);
+    // increase sleep for final retry
+    const sleepDur =
+      retries == MAX_RETRIES
+        ? TASK_FINAL_RETRY_DELAY
+        : retries * TASK_RETRY_BASE_DELAY;
```

Comment:
The precedence here is certainly what we expect, right? As in it won't be `TASK_FINAL_RETRY_DELAY * TASK_RETRY_BASE_DELAY` on the last retry 😂

Reply:
I had the same thought haha! I added in some brackets but then the prettier command removes them. But yes I've tested and it's all good